bevy_lint/src/groups.rs

```diff
@@ -0,0 +1,66 @@
+use crate::LintGroup;
+use rustc_lint::Level;
+
+/// A group of deny-by-default lints that check for outright wrong or useless code.
+///
+/// These lints are carefully picked to be free of false-positives. You should avoid
```

Comment:
```suggestion
/// These lints are carefully picked to be free of false positives. You should avoid
```
Speaking of pedantic :laughing: no hyphen when used as a noun.

Reply:
Hah, good catch!